packages/moonstone/Item/Item.js

```diff
@@ -79,13 +78,19 @@ const ItemBase = kind({
  * @public
  */
 const ItemDecorator = compose(
-	Pure,
 	UiItemDecorator,
 	Spottable,
 	MarqueeDecorator({invalidateProps: ['inline', 'autoHide']}),
 	Skinnable
 );
 
+const ItemFull = ItemDecorator(ItemBase);
+const ItemLightDecorator = compose(
+	Spottable,
```

Comment:
Since `UiItemDecorator` is not wrapped, click/touch will not work until updated.